lib/ajv.js

```diff
@@ -259,8 +259,17 @@ function Ajv(opts) {
 
 
     function addInitialSchemas() {
-        if (self.opts.meta !== false)
-            addMetaSchema(require('./refs/json-schema-draft-04.json'), META_SCHEMA_ID, true);
+        if (self.opts.meta !== false) {
+            var metaSchema = require('./refs/json-schema-draft-04.json');
+
+            addMetaSchema(metaSchema, META_SCHEMA_ID, true);
+
+            // Add other schemas according to JSON specification:
+            // http://json-schema.org/latest/json-schema-core.html
+            addMetaSchema(metaSchema, 'http://json-schema.org/schema', true);
+            addMetaSchema(metaSchema, 'http://json-schema.org/hyper-schema', true);
```

Comment:
You're right, feel free to remove the `hyper-schemas`. By the way, maybe you could make a note about `draft-03` compatibility in the README? I tried it (it was a pipe dream), but it failed to compile or work. I ended up using `json-schema-compatibility` to get it to work instead (https://github.com/mulesoft-labs/osprey-method-handler/blob/master/osprey-method-handler.js#L301-L305)